README.md

```diff
@@ -1,109 +1,88 @@
 
-## Pre-requisites
+## 📖 About
 
-Compatibility tool will work on any modern 64-bit Linux distribution.
-All packaged games are compiled for Steam Runtime Sniper environment and any additional dependencies are provided in the download.
+### 🎨 Features
 
-**Using Luxtorpeda with [Steam native runtime](https://wiki.archlinux.org/index.php/Steam/Troubleshooting#Steam_native_runtime) may or may not work, but is not supported.**
+- 🐧 Modern Native Linux engines
+- :godmode: Better performance and less bugs
+- 🥇 Modern features
+- 🌟 Improved graphics
+- 🐤 Easy to use with automatic installation and configuration
+- 📖 Free and Open Source
+- 👨‍👨‍👧‍👦 Community built and supported
 
-## Installation (using tarball)
+<sub><sup>* All features are dependent on the engine, not all engine have all features.</sub></sub>
 
-*The packages.json for the supported packages and getting the latest versions will get updated on each launch of luxtorpeda, without any input needed from the user. New releases of the luxtorpeda client will need to be downloaded manually, but a new release of the client is not required when a package is created or updated, unless that package depends on a new feature (which will be noted in the release notes).*
+### 😎 Seamless Setup
 
-1. Close Steam.
-2. Download latest version at https://github.com/luxtorpeda-dev/luxtorpeda/releases
-3. Move and unpack tarball to compatibilitytools.d directory (create one if it does not exist):
+Setting up Luxtorpeda is a breeze – simply force your game to use Luxtorpeda through Steam's interface. Once configured, Luxtorpeda seamlessly handles the rest, ensuring a smooth gaming experience with minimal effort on your part. It's like having a dedicated Linux gaming companion that takes care of the technicalities, leaving you to immerse yourself in the gaming world.
 
-    ```bash
-    $ cd ~/.local/share/Steam/compatibilitytools.d/ || cd ~/.steam/root/compatibilitytools.d/
-    $ tar xJf luxtorpeda-<version-number>.tar.xz
-    ```
+![screenshot-0](https://user-images.githubusercontent.com/3967/61964568-7b674500-afce-11e9-9c42-ef6cc1b425b6.png)
 
-4. Start Steam.
-5. In game properties window select "Force the use of a specific Steam Play
-   compatibility tool" and select "Luxtorpeda".
-   
-## Installation (using GUI)
+Discover the future of Linux gaming with Luxtorpeda – your gateway to an elevated, open-source gaming universe.
+
+### ✅ Supported Games
+
+**[Supported Games Webpage](https://luxtorpeda-dev.github.io)**: check out our supported games on our webpage.
+
+**[Supported Games Steam curator](https://store.steampowered.com/curator/41650678/)**: check out our supported games on our Steam curator page.
```

Comment:
I'm not sure who is maintaining this, but it's not something I have control over, so it may not make too much sense to advertise it here.

Reply:
Well it's me 🙂 I think I added you as a administrator as well, let me check